### ISSUE TITLE
indentation error; comment at top of file cause first line to be messed up

### ISSUE THREAD

**Issue by davidmaxwaterman** [PolymerLabs/polyup#25]
Here is a simple polymer element :

```
<link rel="import" href="../bower_components/some-element.html">

<polymer-element name="my-element">
  <template>
    <style>
    </style>
  </template>
  <script>
    Polymer({});
  </script>
</polymer-element>
```

which results in :

```
<link rel="import" href="../bower_components/some-element.html">

<dom-module id="my-element">
  <style>
    </style>
  <template>
  </template>
</dom-module>
<script>
  Polymer({ is: 'my-element' });
</script>
```

1) Note that error in indentation for the closing style tag

More significantly, if you add a comment at the top of the file:

```
<!-- -->
<link rel="import" href="../bower_components/some-element.html">

<polymer-element name="my-element">
  <template>
    <style>
    </style>
  </template>
  <script>
    Polymer({});
  </script>
</polymer-element>
```

you get :

```
<!-- --><html><head><link rel="import" href="../bower_components/some-element.html">

<dom-module id="my-element">
  <style>
    </style>
  <template>
  </template>
</dom-module>
<script>
  Polymer({ is: 'my-element' });
</script>
```

2) Notice the mess that is made of the first line, including 'joinin' it with the second line, and `<html><head>` is added.

**Comment by rictic** [comment]
Sorry for the slow response on this bug, and thanks for taking the time to fill out a detailed bug report, that's appreciated.

While this is a legit bug, it's not one that I'm interested in solving for polyup at this time. The output code is unsightly, but not AFAICT likely to produce invalid results when interpreted by a browser, and someone reading it is likely to notice what's wrong and remove it.

It's expected that the output of polyup will need some polishing in almost all cases.

If this code does fail or cause problems in browsers let me know and we can revisit this.

**Comment by davidmaxwaterman** [comment]
Fair enough. Thanks for considering it.

**Comment by davidmaxwaterman** [comment]
I should note that polyup handles comments poorly in general, including duplicating them, and moving them from on-their-own-line to the end-of-a-line-of-code, and vice-versa.
Lots of editing is required, but at least there's a pattern to watch out for.

**Comment by rictic** [comment]
The duplicated comment bug should be fixed in 0.4.2: #9

It will move comments around though in some cases. If you find any cases where it removes them outright we'd consider that a bug.

**Comment by davidmaxwaterman** [comment]
Yes, that definitely looks like the same problem, thanks.
